src/views/Dashboard/index.tsx

```diff
@@ -49,10 +51,13 @@ export const Dashboard: React.FC<{}> = () => {
       balances?.data?.filter(asset => !asset.balance.isZero()) ?? []
     ).map(a => ({ ...a, backingReserve: a.reserve }));
     return assets.map(asset => {
-      return asset.backingReserve && asset.backingReserve.symbol === "WXDAI"
+      return asset.backingReserve.symbol === nativeSymbols.wrappednative
```

Comment:
Preserves prior nullability semantics.
```suggestion
      return asset.backingReserve?.symbol === nativeSymbols.wrappednative
```